codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/utils/GoCodegenUtils.java

```diff
@@ -225,4 +224,30 @@ public static boolean shapeShouldHaveConversionFunction(Shape shape) {
     }
     return false;
   }
+
+  public static void importAwsSDKShape(
```

Comment:
AWS sdk shapes are either in https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/dynamodb/types or in https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/dynamodb.